File: test13/DBTest/app/src/main/java/ddwucom/mobile/dbtest/MainActivity.java
package ddwucom.mobile.dbtest;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    FoodDBHelper dbHelper;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        dbHelper = new FoodDBHelper(this);
        dbHelper.getReadableDatabase(); //이 순간 테이블이 만들어짐(onCreate 1회 호출)
                                        //이 순간 테이블의 구조를 변경할 수 없음. 내용의 변경이 생기면 앱을 제거해야함
    }
}

File: test12/Exam01/app/src/main/java/ddwucom/mobile/exam01/MainActivity.kt
package ddwucom.mobile.exam01

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    var editText: EditText
    var button: Button!

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        editText = findViewById(R.id.editText)
        button = findViewById(R.id.button)


        button.setOnClickListener(View.OnClickListener {
            val intent = Intent(this@MainActivity, SubActivity::class.java)
            intent.putExtra("text", editText.getText().toString())
            startActivity(intent)
        })
    }
}